Wise_Owl_Library/Models/Author.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Wise_Owl_Library.Models
+{
+    public class Author
+    {
+        public int Id { get; set; }
+
+        [Required(ErrorMessage = "Name is required.")]
+        [StringLength(50, ErrorMessage = "Name length can't be more than 50 characters.")]
```

Comment:
Podle mě tahle validace nepokryje případ kdy tam bude empty nebo whitespace - nevím jestli to přes artributy jde. Já na to používám Custom mappery kde mám pak vše na jednom místě